OpenIPSL/Data/PowerPlant/PFData/BusData.mo

```diff
@@ -0,0 +1,156 @@
+within OpenIPSL.Data.PowerPlant.PFData;
+package BusData "Records with power flow solutions for buses"
```

Comment:
This and the other Data packages break the rule on one file per class. Here several records are placed into one file. Please convert to one-file per class structure.

Reply:
That's right! Fixed in [commit 0bf8ff3](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/0bf8ff3e025a8bddd99fbafd7be0a90fd8530b68).